docassemble/clcsa-01-service-directory/data/questions/main.yml

```diff
@@ -27,77 +27,7 @@ metadata:
       </div>
     </div>
   exit label: ""
-  navigation bar html: |
-
-    <!-- Adds Safe Exit button  -->
-    <!-- Based on https://docassemble.slack.com/files/UHCKKPY3D/F013TPA5000/escape.yml -->
-    <a id="escape" href="https://www.google.com" class="btn btn-warning btn-da">Safe Exit</a>
-    <script>
-      var doc = document;  // Need space!
-
-      var outer = outer || doc.createElement( 'div' );
-      var inner = inner || doc.createElement( 'div' );
-      outer.appendChild( inner );
-
-      outer.setAttribute( 'class', 'escape-outer' );
-      inner.setAttribute( 'class', 'escape-inner' );
-
-      // Give a unique id to each escaper so we can add only
-      // one per outer/inner pair
-      var suffix = suffix || 1;  // Use `suffix` if it already exists
-      var unique_id = 'escaper_' + suffix;
-      var escaper = doc.getElementById( 'escape' );
-      escaper.setAttribute( 'id', unique_id );
-      inner.appendChild( escaper );
-      
-      if (suffix !== 1) {
-        escaper.classList.add( 'escape-overlap' );
-      }
-      suffix += 1;
-      
-      // Add to navbar
-      var after = doc.getElementById('danavbar-collapse');
-      after.insertAdjacentElement('afterEnd', inner);
-    </script>
-    
-    <style>
-      .escape-inner {
-        position: relative;
-        margin-left: .4em;
-      }
-    
-      #escaper_2, .escape-overlap {
-        position: absolute;
-        top: 0;
-        left: 0;
-      }
-      
-      .escape {
-        margin: 0;
-      }
-      
-      @media only screen and (max-width: 30em) {
-        .navbar-brand {
-          margin-right: .3rem;
-        }
-        
-        #dapagetitle {
-          margin-right: .4em;
-        }
-        
-        #dapagetitle .ma_icon {
-          margin-right: 0;
-        }
-        
-        .escape-inner {
-          margin-left: .6em;
-        }
-        
-        #dahelptoggle {
-          padding: 0;
-        }
-      }
-    </style>
+  exit url:  https://clcsa.org.au
```

Comment:
As per #20.

```suggestion
  exit label: 'Safe Exit'
  exit url: 'https://www.google.com'
  exit link: exit
```

Reply:
I don't want to use the `exit *` tags as when the app is in mobile mode the 'Safe Exit' is hidden behind the hamburger menu.   I saw your comments in #20 and am considering them.  I also put a question on the DA Slack and I want to see what comes back from there.

Reply:
DA doesn't have a way to get rid of Exit.  I will implement your suggestion in #20 and fix the CSS as per your other suggested change.

Reply:
If you implement #20, the CSS is no required. The only reason I'm endorsing the approach in #20 is the special behaviour the built-in exit feature has: it deletes the session and answer data (and logs the user out if needed). This aligns more closely to the expected behaviour of a "safe" exit. It is a button clicked in urgency with the intention of removing any traces. Having a normal link would only redirect to another page, but the progress and user answers would still be persisted.

Reply:
Implemented as suggested.